applications/nrf_desktop/configuration/nrf52810dmouse_nrf52810/prj.conf

```diff
@@ -96,7 +96,9 @@ CONFIG_BT_MAX_PAIRED=2
 CONFIG_BT_ID_MAX=3
 CONFIG_BT_LL_SW_SPLIT=y
 
-CONFIG_BT_CONN_TX_MAX=4
+CONFIG_BT_BUF_EVT_RX_COUNT=4
+CONFIG_BT_BUF_ACL_TX_COUNT=3
+CONFIG_BT_CONN_TX_MAX=3
```

Comment:
I guess that your intention here is to reduce RAM usage next to setting `CONFIG_BT_BUF_EVT_RX_COUNT=4`.  I am slightly worried that the configuration changes might introduce edge cases here.

Reply:
Updates in dongle's configuration might require additional testing too

Reply:
I see support for 52810 was dropped here: https://github.com/nrfconnect/sdk-nrf/commit/f66256c3ba4ccc63c391cf264265b28a0de15d6d

Is this a concern for the v2.6.4-x? this is a preview tag.